samples/csharp/getting-started/Forecasting_BikeSharingDemand/README.md

```diff
@@ -0,0 +1,104 @@
+
+The original dataset contains several columns corresponding to seasonality and weather. For brevity and because the technique used in this sample only requires the values from a single numerical column, the original dataset has been enhanced to include only the following columns:  
+
+- **dteday**: The date of the observation.
+- **year**: The encoded year of the observation (0=2011, 1=2012).
+- **cnt**: The total number of bike rentals for that day.
+
+The original dataset is mapped to a database table with the following schema in a SQL Server database.
+
+```sql
+CREATE TABLE [Rentals] (
+	[RentalDate] DATE NOT NULL,
+	[Year] INT NOT NULL,
+	[TotalRentals] INT NOT NULL
+);
+```
+
+The following is a sample of the data:
+
+| RentalDate | Year | TotalRentals |
+| --- | --- | --- |
+|1/1/2011|0|985|
+|1/2/2011|0|801|
+|1/3/2011|0|1349|
+
+## Database Loader
+
+Database Loader provides a simple API to read data from relational databases directly into an `IDataView`. This loader supports any relational database provider supported by System.Data in .NET Core or .NET Framework, meaning that you can use any RDBMS such as SQL Server, Azure SQL Database, Oracle, SQLite, PostgreSQL, MySQL, Progress, IBM DB2, etc.
+
+To load data, you need to provide a connection string and a SQL command to get data from the database.
+
+```csharp
+string rootDir = Path.GetFullPath(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "../../../"));
+string dbFilePath = Path.Combine(rootDir, "Data", "DailyDemand.mdf");
+var connectionString = $"Data Source=(LocalDB)\\MSSQLLocalDB;AttachDbFilename={dbFilePath};Integrated Security=True;Connect Timeout=30;";
+
+MLContext mlContext = new MLContext();
+DatabaseLoader loader = mlContext.Data.CreateDatabaseLoader<ModelInput>();
+
+string query = "SELECT RentalDate, CAST(Year as REAL) as Year, CAST(TotalRentals as REAL) as TotalRentals FROM Rentals;";
+
+DatabaseSource dbSource = new DatabaseSource(SqlClientFactory.Instance,
+                                            connectionString,
+                                            query);
+
+IDataView dataView = loader.Load(dbSource)
+```
+
+## ML task - [Regression](https://docs.microsoft.com/en-us/dotnet/machine-learning/resources/tasks#regression)
+
+The ML Task for this sample is forecasting, which is a supervised machine learning task that is used to predict the value of the label (in this case the demand units prediction) from previous data.
+
+## Solution
+
+To solve this problem, you build and train an ML model on existing training data, evaluate how good it is (analyzing the obtained metrics), and lastly you can consume/test the model to predict the demand given input data variables.
+
+![Build -> Train -> Evaluate -> Consume](../shared_content/modelpipeline.png)
+
```

Comment:
Shouldn't you provide some info about the actual pipeline and algorithm after loading the data and before showing the predictions output?